app/src/main/res/values/fetcharr.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="post_trip_card">
+        <item> Name\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\tSeats\nRating</item>
```

Comment:
Not sure of the purpose of this code, but hard coding tabs to separate fields is a smell that probably indicates a larger problem. Is there a way you could split up actual text fields, etc. and then position them appropriately such that you wouldn't need this tabbed implementation?

Reply:
That one is code from an alternate implementation, it looks like we forgot to delete the xml.

Reply:
Sweet, thanks for the clarification!